src/coreComponents/schema/docs/FaceManager_other.rst

```diff
@@ -1,25 +1,25 @@
-neighborData            node                                                                                                                                                                                  :ref:`DATASTRUCTURE_neighborData`                         
-sets                    node                                                                                                                                                                                  :ref:`DATASTRUCTURE_sets`                                 
-======================= ==================================================================================== ================================================================================================ ========================================================= 
+======================= ================================================================================== ================================================================================================ ========================================================= 
+Name                    Type                                                                               Registered By                                                                                    Description                                               
+======================= ================================================================================== ================================================================================================ ========================================================= 
+domainBoundaryIndicator integer_array                                                                                                                                                                       (no description available)                                
+edgeList                geosx_InterObjectRelation< LvArray_ArrayOfArrays< int, int, LvArray_ChaiBuffer > >                                                                                                  (no description available)                                
+elemList                integer_array2d                                                                                                                                                                     (no description available)                                
+elemRegionList          integer_array2d                                                                                                                                                                     (no description available)                                
+elemSubRegionList       integer_array2d                                                                                                                                                                     (no description available)                                
```

Comment:
A lot of these changes are undesirable. On the other hand, it's impossible to distinguish aliases that map to the same type. Not sure if there is a solution to this.

Reply:
I didn't look at the rst changes...I don't understand why these have propagated to the rst generation.

Reply:
The XML/RST name is derived from `typeid`, but now `typeid(integer_array2d) == typeid(localIndex_array2d)` so I guess it just picks the first one it finds